### DISCUSSION TITLE
FirefoxPWA 1.1.0

### DISCUSSION THREAD

**Discussion by filips123** [filips123/PWAsForFirefox#54] [Announcements]
## Important

* Original Homebrew formula had a bug that prevented PWAs from launching on macOS after updating the formula. I reuploaded the formula with a fix. To install the fixed version, run:

  ```bash
  brew update # To update all taps
  brew reinstall firefoxpwa # To reinstall FirefoxPWA with a fix
  ```

  Then, edit/update all existing PWAs through extension or command-line program to apply the fix to PWAs. Alternatively, you can remove and reinstall your PWAs.

## Added

* **Support for reinstalling runtime from extension (#47):**
  You can now reinstall/update the Firefox runtime from the settings page of the extension.

* **Theme color-based transparent tabs (#48):**
  Tabs are now slightly transparent/use a shade of theme color, so they should not look so bad with some PWAs.

* **Customize option to disable the title bar (#49):**
  You can now completely disable the title bar, including all icons. Instructions are in [the repository wiki](https://github.com/filips123/FirefoxPWA/wiki/App-Browser-Features). This can be an unstable feature and is only meant for tiling window manager users!

## Changed

* Display all contributors and sponsors in READMEs and download page.

## Fixed

* Fix auto publishing scripts.
* Fix AUR binary install directory.

<hr /><em>This discussion was created from the release <a href='https://github.com/filips123/FirefoxPWA/releases/tag/v1.1.0'>FirefoxPWA 1.1.0</a>.</em>

**Comment by black7375** [comment]
I updated to the latest version, but I can't type in the URL Bar. Is this an intentional change?

**Environment**
Manjaro KDE

**Reproduce Steps**
Application Menu - Settings - Click to URL Bar

**Reply by filips123** [reply]
It is intentional that the URL bar is read-only (although I think I already implemented this before 1.1.0), because it is mainly meant as URL indicator so you know when you are out of PWA scope, and because it caused some problems with other UI modifications I did. However, you can press `F6` and then manually enter URL you want using the popup.

**Reply by black7375** [reply]
I found out this time because I was using the old version and recently updated it.

It was a little inconvenient that it was not possible to input directly from the URL bar.
I thought it was non-intuitive and necessary when performing actions like installing add-ons.
Of course, it's not a huge problem.

Thank you for your reply.

**Reply by filips123** [reply]
I could probably add an option to enable URL bar input again but unfortunately I currently don't have much time.

**Reply by black7375** [reply]
It's okay.
You don't have to be burdened.

It doesn't matter if you do it slowly later when you have time.
